lib/js/src/protocol/_SdlProtocolBase.js

```diff
@@ -406,6 +435,23 @@ class _SdlProtocolBase {
         const version = sdlPacket.getVersion();
         const serviceType = sdlPacket.getServiceType();
         if (version >= 5) {
+            // check if vehicleType data exists then fire onVehicleTypeReceived protocol listener event
+            const vehicleType = this._getVehicleType(sdlPacket);
+            if (
+              vehicleType
+              && this._sdlProtocolListener !== null
+              && typeof this._sdlProtocolListener.onVehicleTypeReceived === 'function'
```

Comment:
The indentation for lines 441-443 is incorrect. There is an indentation of 14 spaces but it should be 16. Use the command `npm run lint` to confirm.

Reply:
Fixed